### DISCUSSION TITLE
Release 0.96.2.0for1.12.2 seems have wrong sources

### DISCUSSION THREAD

**Discussion by hohserg1** [TownyAdvanced/Towny#6340] [Q&A]
https://github.com/TownyAdvanced/Towny/releases/tag/0.96.2.0for1.12.2
If you open english.yml from jar, you can see `version: 0.83`
If you open english.yml from sources, you can see `version: 0.47`
So, seems commit 6839346 is not `0.96.2.0 for 1.12.2` version

**Comment by LlmDl** [comment]
You might be right, but no one should be using that version on 1.12.2: I backported Towny for MC 1.12.2 when 0.97.0.0 was released: https://github.com/TownyAdvanced/Towny/releases/tag/0.97.0.0for1.12.2
